src/lib/serviceRoutinesV2/postEntity.cpp

```diff
@@ -100,14 +100,14 @@ std::string postEntity
     }
   }
 
-  // Default value for status code: SccCreated
+  // Default value for status code: SccNoContent
   if ((ciP->httpStatusCode == SccOk) || (ciP->httpStatusCode == SccNone) || (ciP->httpStatusCode == SccCreated))
```

Comment:
Fixed in 56c9201.

I have also removed the SccNone comparison at the end (having a SccNone at this point would be an internal fatal error that I prefer not to mask/shadow). Harness tests run ok after the change.